curp/proto/message.proto

```diff
@@ -89,11 +89,35 @@ message ShutdownResponse {
     optional bytes error = 3;
 }
 
+
+
+message ProposeConfChangeRequest {
+    enum ConfChangeType {
+        Add = 0;
+        AddLearner = 1;
+        Remove = 2;
+        Update = 3;
+    }
+    message ConfChange {
+        ConfChangeType change_type = 1;
+        uint64 node_id = 2;
+        string address = 3;
+    }
+    string id = 1;
+    repeated ConfChange changes = 2;
```

Comment:
When a node can have multiple addresses, does this mean using multiple ConfChange for the same node_id?

Reply:
No, we need no add `repeated` to address if  a node can have multiple addresse. 
Added